### ISSUE TITLE
Tensor dimension

### ISSUE THREAD

**Issue by xiazhiwang-zz** [wangyida/softpool#2]
Thanks for your excellent work!

When I run the code, I got error on https://github.com/wangyida/softpool/blob/cd8f414af11cc4fb9cf95b726210b1f61418e3bd/tensorflow/models/pcn_reg.py#L175. It seems that gt has 3 dimensions but you select the 4th dimension.

Could you please help this?

**Comment by wangyida** [comment]
Hi Xiazhi, I think your data does not hace the color dimentionality, the fourth channel could be used as visualization or semantic labels. I use it to render 3D model in category related colors. 
You'd better try our PyTorch model while Tensorflow has been updated to version 2 while our codes are delivered in version 1
